lib/logstash/outputs/sqs.rb

```diff
@@ -98,9 +102,12 @@ def register
    end
 
     begin
-      @logger.debug('Connecting to SQS queue', :queue => @queue, :region => region)
-      @queue_url = @sqs.get_queue_url(:queue_name => @queue)[:queue_url]
-      @logger.info('Connected to SQS queue successfully', :queue => @queue, :region => region)
+      params = { queue_name: @queue }
```

Comment:
I think this closely resembles Chris' suggestion, but I felt it was a good balance of simplicity and fulfilling requirements. Note I did change the logging message to match the exact AWS attribute "queue_name" over the previously used "queue".